resources/hpxml-measures/BuildResidentialHPXML/resources/geometry.rb

```diff
@@ -1278,9 +1278,44 @@ def self.add_windows_to_wall(surface, window_area, window_gap_y, window_gap_x, w
     window_width = Math.sqrt((window_area / num_windows.to_f) / window_aspect_ratio)
     window_height = (window_area / num_windows.to_f) / window_width
     width_for_windows = window_width * num_windows.to_f + window_gap_x * num_window_gaps.to_f
+
     if width_for_windows > wall_width
-      runner.registerError("Could not fit windows on #{surface.name}.")
-      return false
+      surface_area = UnitConversions.convert(surface.grossArea, 'm^2', 'ft^2')
+      runner.registerWarning("Surface #{surface.nameString} has an area of #{surface_area}, params: window_area=#{window_area} ft^2, window_gap_y=#{window_gap_y} ft, window_gap_x=#{window_gap_x} ft, window_aspect_ratio=#{window_aspect_ratio}, max_single_window_area=#{max_single_window_area} ft^2, facade=#{facade}")
+      runner.registerWarning("Could not fit windows on #{surface.name}. Fall back to WWR")
+      wwr = window_area / surface_area
+      if wwr >= 0.90
+        wwr = 0.90
```

Comment:
this case seems like it needs an additional warning that we're not just falling back on the WWR - we're actually ignoring it and falling back to a maximum WWR
```suggestion
      max_wwr = 0.9
      if wwr >= max_wwr
        runner.registerWarning("WWR too high (#{wwr}) for #{surface.name}. Falling back to reasonable maximum WWR #{max_wwr}")
        wwr = max_wwr
```

Reply:
Sure, doesn't hurt.  Note that I didn't add it because:
* Either you don't care, because it has no impact overall since you managed to add more window area to another wall for that facade
* Or you do care, but you'll get https://github.com/T110-nrgx/resstock/blob/09c97b782c22518be604cda94359d4826ac42d06/resources/hpxml-measures/BuildResidentialHPXML/resources/geometry.rb#L1158

Reply:
(Generally speaking, logging is cool and all. In the OpenStudio ecosystem, when you have millions of lines of code being visited for a single run, you kinda have to start thinking carefully about whether you should really log something and at which severity, because having 1000s of log lines for a normal run means there's a chance of just drowning any relevant info)

Reply:
I just saw that on my upstream PR I got the same exact feedback about adding a comment, so I guess we really do need one :)